### PR TITLE
Add test for group decrypt vectors

### PR DESCRIPTION
**Pull request by Powersource**
For https://github.com/ssbc/ssb-tribes/issues/102

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "ssb-meta-feeds": "~0.38.0",
     "ssb-query": "^2.4.5",
     "ssb-tribes": "^2.7.4",
+    "ssb-tribes-vectors": "github:ssbc/ssb-tribes",
```
Comment (Powersource) [inline package.json:51]: uninstall

File: test/index.js [hunk 2]
```diff
@@ -352,3 +353,40 @@ test('encrypt accepts keys as recps', (t) => {
     t.end()
   })
 })
+
+test('decrypt group vectors', async (t) => {
+  const vectors = [
+    require('ssb-tribes-vectors/test/vectors/unbox1.json'),
+    require('ssb-tribes-vectors/test/vectors/unbox2.json')
```
Comment (Powersource) [inline test/index.js:360]: move these to envelope-spec and import them from there instead

File: package.json [hunk 3]
```diff
@@ -26,6 +26,7 @@
   },
   "devDependencies": {
     "c8": "^7.11.0",
+    "envelope-spec": "github:ssbc/envelope-spec#group-test-vectors",
```
Comment (Powersource) [inline package.json:29]: https://github.com/ssbc/envelope-spec/pull/23

**Review by mixmix — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (mixmix) [reply package.json:29]: :fire: these vectors should be moved to `private-group-spec`, see PR

File: package.json [hunk 4]
```diff
@@ -32,6 +32,7 @@
     "mkdirp": "^1.0.4",
     "prettier": "^2.6.2",
     "pretty-quick": "^3.1.3",
+    "private-group-spec-vectors": "github:ssbc/private-group-spec#add-classic-unbox-vectors",
```
Comment (Powersource) [inline package.json:35]: https://github.com/ssbc/private-group-spec/pull/36

Comment (Powersource) [inline package.json:35]: added this as an extra dep, to not have to update `private-group-spec` initially. let's see if we can update it without too many breaking changes

Comment (Powersource) [reply package.json]: it seems we weren't using the spec for anything, so merged the deps now

File: package.json [hunk 5]
```diff
@@ -32,6 +31,7 @@
     "mkdirp": "^1.0.4",
     "prettier": "^2.6.2",
     "pretty-quick": "^3.1.3",
+    "private-group-spec": "github:ssbc/private-group-spec#add-classic-unbox-vectors",
```
Comment (Powersource) [inline package.json:34]: https://github.com/ssbc/private-group-spec/pull/36

File: package.json (hunk 4 above)
Comment (Powersource) [reply package.json]: oops lol i searched for the wrong thing. but tests seem to be passing anyway

File: package.json (hunk 5 above)
Comment (Powersource) [reply package.json:34]: v8.1.0

**Pull request merged**